data/Makefile

```diff
@@ -87,70 +101,139 @@ endif
 .DELETE_ON_ERROR:
 .SECONDARY:
 
-.PHONY: f1chr4 f1chr2R f1 fishchr25 fish
+.PHONY: f1chr4 f1chr2R f1 fishchr25 fish physical-map scaffolds
 all: f1chr4 f1chr2R f1 fishchr25 fish
 
+$(lr).physlr.physical-map: $(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.path
+	ln -sf $< $@
+
+$(lr).physlr.physical-map.$(ref).n10.paf.gz: $(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.map.$(ref).n10.paf.gz.pdf
+	ln -sf $< $@
+
+$(draft).physlr.fa: $(lr).k$k-w$w.n$(n1)-$(n2).c2-$(minimizer_multiplicity).physlr.overlap.m$m.mol.mol2-bcs.backbone.map.$(draft).n10.sort.best.bed.path.fa
+
+scaffolds:
+
+ifndef ref
+scaffolds: \
+	physical-map \
+	$(draft).physlr.fa
+else
+scaffolds: \
+	physical-map \
+	$(draft).physlr.quast.tsv
+endif
+
+physical-map:
+
+ifndef ref
+physical-map: \
+	$(lr).physlr.physical-map
+else
+physical-map: \
+	$(lr).physlr.physical-map \
+	$(lr).physlr.physical-map.$(ref).n10.paf.gz.pdf
+endif
+# Help
+help:
+	@echo "Usage: ./make [COMMAND] [OPTION=VALUE]..."
+	@echo "    Commands:"
+	@echo ""
+	@echo "	physical-map		run physlr and generate a physical map with linked reads (10X or stLFR)"
+	@echo "	scaffold	        run physlr and scaffold an existing assembly with a physical map"
+	@echo "	help                    display this help page"
+	@echo "	version                 display the software version"
+	@echo ""
+	@echo "    Options:"
+	@echo "	draft               draft name [draft]. File must have .fa extension"
+	@echo "	ref                 reference name [ref]. File must have .fa extension. If provided, will generate Quast metrics and backbone visualization relative to reference."
+	@echo "	lr                  read name [reads]. File must have .fq.gz extension"
+	@echo "	t		    number of threads used [16]"
+	@echo "	k		    size of k-mers [40]."
+	@echo "	w		    window size to choose minimizers [32]."
+	@echo "	minimizer_overlap		% of edges to remove in the overlap graph [90]. Available options are stLFR, 10X, or [0-100]."
+	@echo "					Default values for stLFR and 10X were obtained by finding a value that work well across multiple datasets"
```

Comment:
I think I was unclear here -- I mean mention what the default values are for stLFR and 10x